src/openzeppelin/account/library.cairo

```diff
@@ -1,334 +1,240 @@
-# SPDX-License-Identifier: MIT
-# OpenZeppelin Contracts for Cairo v0.3.2 (account/library.cairo)
+// SPDX-License-Identifier: MIT
```

Comment:
i think the changes in this PR are breaking enough to drop https://github.com/OpenZeppelin/cairo-contracts/pull/446 and just slide the camelCase migration in here, not even the doc changes will apply.